src/test/java/edu/kit/ipd/crowdcontrol/objectservice/payment/PaymentCalculatorTest.java

```diff
@@ -95,7 +98,7 @@ public void testEstimatePayment() throws Exception {
 
         sortedResult = sortWorkerMap(resultRatings);
 
-
+        workerRecordList.add(workerThree);
```

Comment:
Because SortedMap.get(sortedMap.lastKey()) is needed for an assertion and workerThree is not part of that calculation